composer.json

```diff
@@ -17,6 +17,7 @@
         "paragonie/random_compat": "~1.0|~2.0"
     },
     "require-dev": {
+        "phpunit/phpunit": "~5.4",
```

Comment:
This is an issue as PHPUnit 5.4 requires PHP 5.6 while this bundle supports PHP 5.3.